src/openms_gui/include/OpenMS/VISUAL/LayerDataFeature.h

```diff
@@ -70,6 +70,14 @@ namespace OpenMS
       return getFeatureMap()->getUnassignedPeptideIdentifications();
     }
 
+    virtual void setPeptideIds(const PepIds& ids) override
+    {
+      getFeatureMap()->getUnassignedPeptideIdentifications() = ids;
```

Comment:
Is this what we want here? Only the unassigned?

Reply:
this is only a refactoring.
And probably, the answer is `no`, but to also assign IDs to features one would need some IDMapping parameters. And I guess they are hard to guess, and thus would need to taken from the stored parameters in the featuremap, if they exist, or queried from the user... and I guess all of this was a bit too much when this was originally implemented...